src/types/auditLog.ts

```diff
@@ -16,4 +16,5 @@ export type AuditLog = {
   actor: string
   page: string
   remarks: string
+  link?: string
```

Comment:
why is this optional ah? isnt this available for all commits?

Reply:
Hmm this one is for forward compatibility, because not every audit log item will have a relevant link. For instance, when we support adding/removing collaborators, we won't be able to provide a link for this, so I opted to keep it optional for now.